data/human/free worlds 0 prologue.txt

```diff
@@ -1280,7 +1280,7 @@ mission "FW Katya 1"
 					decline
 			
 			label terrorists
-			`	"Look, kid," she says. "I know the situation looks bad, but I'm telling the truth when I say that the Free Worlds aren't the ones who dropped those nukes, and we don't know who did, either. So I'll proudly be called treasonous for separating from the Republic, but I will not tolerate being called a terrorist."`
+			`	"Look, kid," she says. "I know the situation looks bad, but I'm telling the truth when I say that the Free Worlds isn't the one who dropped those nukes, and we don't know who did, either. So I'll proudly be called treasonous for separating from the Republic, but I will not tolerate being called a terrorist."`
```

Comment:
This seems to refer to the people within the Free Worlds.
If it were about the Republic, I still think it would be plural.

Reply:
Maybe:
```suggestion
			`	"Look, kid," she says. "I know the situation looks bad, but I'm telling the truth when I say that the Free Worlders aren't the ones who dropped those nukes, and we don't know who did, either. So I'll proudly be called treasonous for separating from the Republic, but I will not tolerate being called a terrorist."`
```